catalog/rest/catalog-rest-endpoint/pom.xml

```diff
@@ -24,7 +24,15 @@
     <artifactId>catalog-rest-endpoint</artifactId>
     <packaging>bundle</packaging>
     <name>DDF :: Catalog :: REST :: Endpoint</name>
+
+    <properties>
+        <pax.exam.version>4.11.0</pax.exam.version> <!-- TODO: Move to root pom file -->
```

Comment:
It's hierarchical inheritance. In other words, you can only use properties that belong to  yourself or a parent of yourself. There are workarounds like importing other poms directly but I'd rather just move it up to root like the todo says.